### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by lvh**

### CODE REVIEW

**Comment by lvh** [comment]
```
lvh@zygalski ~/P/r/reason (upgrayed)> git describe HEAD                 09:34:14
0.8.0-9-gc145a73
lvh@zygalski ~/P/r/reason (upgrayed)> git status                        09:34:15
On branch upgrayed
nothing to commit, working tree clean
lvh@zygalski ~/P/r/reason (upgrayed)> lein figwheel                     09:34:16
Figwheel: Cutting some fruit, just a sec ...
Figwheel: Validating the configuration found in project.clj
------ Figwheel Configuration Error ------

Error at [:cljsbuild :builds 2 :compiler :output-dir]
All ClojureScript build config :output-dir parameters should be unique

  {:cljsbuild
   {:builds
    [{:compiler
      {:main "reason.runner"
       :optimizations :whitespace
       :pretty-print true
       :output-dir ...
       ^----
       }}]}}

-- Docs for key :output-dir --
Sets the output directory for output files generated during
compilation.

Defaults to  "out".

  :output-dir "resources/public/js/out"

------------------------------------------
Figwheel: There are errors in your configuration file - project.clj
Figwheel: Would you like to ...
(f)ix the error live while Figwheel watches for config changes?
(q)uit and fix your configuration?
(s)tart Figwheel anyway?
Please choose f, q or s and then hit Enter [f]:
```

**Comment by lvh** [comment]
@smogg This is now ready to review :)

File: project.clj [hunk 1]
```diff
@@ -4,54 +4,48 @@
...
+      :output-dir "dev-resources/public/js/compiled/"
+      :source-map-timestamp true}}
+    {:id "min"
+     :source-paths ["src"]
+     :compiler
+     {:main "reason.core"
+      :optimizations :advanced
+      :output-dir "target/min/"
+      :pretty-print false}}
+    {:id "test"
```
Comment (smogg) [inline project.clj:37]: we probably want `output-to` in builds (we load reason.js in html but I don't think it will ever be generated)

**Comment by smogg** [comment]
This lgtm modulo comment

File: project.clj (hunk 1 above)
Comment (lvh) [reply project.clj:37]: Good point. I had to remove some of this because figwheel was complaining.

**Comment by smogg** [comment]
Lgtm

**Pull request merged**